src/platformer/platformer.py

```diff
@@ -41,68 +42,118 @@ def update(self):
     # Draw things
     def render(self):
         self.win.fill(BLACK)
-        platforms.render()
+        self.platforms.render()
         self.player.render()
         pg.display.update()
 
     # The main loop
     def loop(self):
         while self.running:
-            self.check_events()
+            self.checkEvents()
             self.update()
             self.render()
             self.clock.tick(FPS)
         pg.quit()
 
 
 class Physics:
+    gravity = 0.5
+    jumpHeight = -12
+    friction = 0.5
+    acceleration = 1
+    maxXVel = 8
 
-    gravity = 2
-    jumpHeight = 18
-    friction = 2
-    acceleration = 4
-
-    @staticmethod
     def squareCollision(x1, y1, w1, h1, x2, y2, w2, h2):
-        return ((x1 + w1 > x2) and (x1 < x2 + w2)) and (
-            (y1 + h1 > y2) and (y1 < y2 + h2)
-        )
+        return x1 + w1 > x2 and x1 < x2 + w2 and y1 + h1 > y2 and y1 < y2 + h2
 
 
 class Player:
     def __init__(self):
-        self.x = 0
-        self.y = 0
-        self.xVel = 0
-        self.yVel = 0
+        self.pos = pg.Vector2(250, 300)
+        self.vel = pg.Vector2(0, 0)
 
     def colliding(self):
+        for platform in main.platforms.rects:
+            if pg.Rect(self.pos.x, self.pos.y, 50, 50).colliderect(platform):
+                return True
         return False
 
     def update(self):
-        self.yVel += Physics.gravity
-        self.y += self.yVel
+        # Gravity
+        keyDown = pg.key.get_pressed()
+        self.vel.y += Physics.gravity
+        self.pos.y += self.vel.y
```

Comment:
.

Reply:
Well, I left Unity before I even modded my first game bc I never knew C#. After I learned a bit of C#, I realised that C# was a less powerful Java lol, so yeah.
I'll be using Git when I get time to learn it since I study everything like 30X slower than you do lol. I code some of the platformer whenever I'm free and I can use my PC.
And OK, I'll be experimenting with dt and then try using it for the physics.

Reply:
np, I am improving physics system

Reply:
So... what now? Special effects? Levels? Graphics (bc squares look kind of boring)?